libraries/GCS_MAVLink/examples/routing/routing.cpp

```diff
@@ -41,6 +41,11 @@ class GCS_MAVLINK_routing : public GCS_MAVLINK
     const AP_FWVersion &get_fwver() const override { return fwver; }
     void set_ekf_origin(const Location& loc) override { };
 
+    uint8_t heartbeat_frame_type() const override { return 1; }
+    uint8_t heartbeat_base_mode() const override { return 2; }
+    uint32_t heartbeat_custom_mode() const override { return 3; }
```

Comment:
why not use the enums here?

Reply:
enum TOMs_USELESS_NUMBER
{ ...

Reply:
Type have also been tightened here, and a comment added about them being dummy values.